.travis.yml

```diff
@@ -5,6 +5,7 @@ php:
   - 7.1
   - 7.2
   - 7.3
+  - 7.4
 install: composer install --no-interaction --prefer-source
```

Comment:
Btw why we’re  using —prefer-source and not —prefer-dist?

Reply:
I believe because we are doing git-based installation. Is there a benefit from using `prefer-dist` option in our context?

Reply:
Yes, installing from source mean that composer will clone all packages as a fit repo, while “dist” means that packages will be tried to be downloaded as zip files exact commit and just unzipped, usually it much faster

Reply:
I just created PR with changing to dust, let’s see if it will work